web/src/Mainpage.tsx

```diff
@@ -0,0 +1,142 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Scrollbar, A11y } from "swiper/modules";
+import { useState } from "react";
+import JuiceComp from "./component/JuiceComp.tsx";
+
+function Mainpage() {
+  const [close, setClose] = useState(true);
+
+  function closeAll() {
```

Comment:
javascript는 함수형 프로그래밍 페러다임을 사용하는데, 그 패러다임에 최적화된 함수 선언 방식이 `익명 함수`야 (anonymous function) 최신 자바스크립트에서 왠만한 함수는 익명 함수를 사용해

```suggestion
const closeAll = () => setClose(!close);
```

이 방식을 쓰면 함수를 좀더 간단하게 관리할 수 있고 함수 자체를 일급 객체로 관리하는 자바스크립트의 특성을 잘 표현할 수 있지

> [!NOTE]
> 코드를 작성할 때는 각 언어의 특징과 페러다임에 따라서 작성하는 것이 좋아